backend/src/lib/controllers/user-ctrl.ts

```diff
@@ -19,6 +20,57 @@ async function getUsers(req: Request, res: Response) {
   }
 }
 
+async function addUsers(req: Request, res: Response) {
+  const user = AuthService.getCurrentUser(req);
+
+  if (!user) {
+    res.status(401);
+    return res.send("Unauthorized");
+  }
+
+  const { role } = req.body;
+  const { emails } = req.query;
+
+  if (!role) {
+    res.status(400).send("Missing required body `role`");
+    return;
+  }
+
+  if (role !== "Admin" && role !== "Editor" && role !== "Publisher") {
```

Comment:
Instead of hardcoding the strings here, you could use the `enum Role` you created. 

```
if (role !== Roles.Admin ...)
```